config/mail.php

```diff
@@ -57,112 +38,29 @@
     | their respective settings. Several examples have been configured for
     | you and you are free to add your own as your application requires.
     |
-    | Laravel supports a variety of mail "transport" drivers to be used while
-    | sending an e-mail. You will specify which one you are using for your
-    | mailers below. You are free to add additional mailers as required.
+    | Laravel supports a variety of mail "transport" drivers that can be used
+    | when delivering an email. You may specify which one you're using for
+    | your mailers below. You may also add additional mailers if needed.
     |
     | Supported: "smtp", "sendmail", "mailgun", "ses", "ses-v2",
-    |            "postmark", "log", "array", "failover", "roundrobin"
+    |            "postmark", "resend", "log", "array",
+    |            "failover", "roundrobin"
     |
     */
 
     'mailers' => [
+
         'smtp' => [
-            // Don't touch this. For the smtp mailer, the transport needs to be smtp
             'transport' => 'smtp',
-
-            /*
-            |--------------------------------------------------------------------------
-            | SMTP Host Address
-            |--------------------------------------------------------------------------
-            |
-            | Here you may provide the host address of the SMTP server used by your
-            | applications. A default option is provided that is compatible with
-            | the Mailgun mail service which will provide reliable deliveries.
-            |
-            */
-            'host' => env('MAIL_HOST', 'smtp.mailgun.org'),
-
-
-            /*
-            |--------------------------------------------------------------------------
-            | SMTP Host Port
-            |--------------------------------------------------------------------------
-            |
-            | This is the SMTP port used by your application to deliver e-mails to
-            | users of the application. Like the host we have set this value to
-            | stay compatible with the Mailgun e-mail application by default.
-            |
-            */
-
-            'port' => env('MAIL_PORT', 587),
-
-            /*
-            |--------------------------------------------------------------------------
-            | SMTP Server Username
-            |--------------------------------------------------------------------------
-            |
-            | If your SMTP server requires a username for authentication, you should
-            | set it here. This will get used to authenticate with your server on
-            | connection. You may also set the "password" value below this one.
-            |
-            */
-
+            'scheme' => env('MAIL_SCHEME'),
+            'url' => env('MAIL_URL'),
```

Comment:
And this also - is that the base URL for images or links in mail, or something else?